test/ScanDependencies/Inputs/unicode_filёnamё.swift

```diff
@@ -0,0 +1,3 @@
+func foo() {
+    print(1)
```

Comment:
Having an actual test itself have a Unicode character in its filename seems to trip up lit test infra.